WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -677,7 +677,7 @@ private extension OrderDetailsViewController {
     func editShippingAddressTapped() {
         let viewModel = EditAddressFormViewModel(siteID: viewModel.order.siteID, address: viewModel.order.shippingAddress)
         let editAddressViewController = EditAddressHostingController(viewModel: viewModel)
-        show(editAddressViewController, sender: self)
+        present(editAddressViewController, animated: true, completion: nil)
```

Comment:
```suggestion
        let navigationController = WooNavigationController(rootViewController: editAddressViewController)
        present(navigationController, animated: true, completion: nil)
```

Now that I think about it, If we wrap the `EditAddressHostingController` we won't require the `ZStack` workaround & we will gain back the custom back button configuration.

What do you think? I think it's worth it.